### ISSUE TITLE
kedro-kubeflow breaked TemplatedConfigLoader

### ISSUE THREAD

**Issue by 0shimax** [getindata/kedro-kubeflow#107]
When I installed just Kedro and set up `hooks.py`, Kedro reads globals.yml. However, when I installed kedro-kubuflow, globals.yml is not read properly.
Is the above behavior what kedro-kubeflow is intended to do?

**Comment by 0shimax** [comment]
This problem was solved by adding the following to `settings.py`.
````
DISABLE_HOOKS_FOR_PLUGINS = ("kedro-kubeflow",)
```

Related issue: https://github.com/getindata/kedro-kubeflow/issues/72

**Comment by 0shimax** [comment]
Am I understanding correctly that `global.yml` is ignored and `KEDRO_CONFIG_{name}` is used when executing `kedro kubeflow compile`? Please let me know.

**Comment by 0shimax** [comment]
I confirmed `kedro kubeflow compile` doesn's use `global.yml`. 
Why do you code
```
    @hook_impl
    def register_config_loader(
        self, conf_paths: Iterable[str]
    ) -> ConfigLoader:
        return TemplatedConfigLoader(
            conf_paths,
            globals_dict=self.read_env(),
        )
```

instead of 

```
    @hook_impl
    def register_config_loader(
        self, conf_paths: Iterable[str]
    ) -> ConfigLoader:
        return TemplatedConfigLoader(
            conf_paths,
            globals_pattern="*globals.yml",
            globals_dict=self.read_env(),
        )
```

**Comment by szczeles** [comment]
Hey @0shimax! Unfortunately, this is true, `global.yml` is ignored when the plugin is installed and the hook is not disabled (as you've seen in #72). I think we need here a better way to handle dynamic parametrization in `kedro_kubeflow/config.py` and abandon the hook idea.

We will think about the solution, in the meantime please disable the hook.

**Comment by szczeles** [comment]
@0shimax This issue is fixed in 0.6.0 release

**Comment by 0shimax** [comment]
@szczeles Thanks for the quick fix!
